client/my-sites/plugins/plan-setup/index.jsx

```diff
@@ -1,81 +1,336 @@
 /**
```

Comment:
The plugins you've just purchased as part of your jetpack plan (VaultPress/Akismet/Polldaddy). Not Jetpack itself, that's already set up if you've made it this far. `jetpack-plan-setup` might also make sense, `jetpack-plugins-setup` if the prefix is not important.